server-common/src/main/java/org/apache/kafka/server/common/TopicIdPartition.java

```diff
@@ -54,3 +63,6 @@ public String toString() {
         return topicId + ":" + partitionId;
     }
 }
+
+
+
```

Comment:
nit: Could we remove those two empty lines? We just need one at the end of the file.